### ISSUE TITLE
Missing System.Windows.Media.Animation.BackEase in WpfGeneratedKnownTypes 

### ISSUE THREAD

**Issue by lindexi** [dotnet/wpf#6588]
The System.Xaml.dll!System.Xaml.Schema.XamlObjectCreationFactory can create object directly by WpfGeneratedKnownTypes  help. But we can not find the System.Windows.Media.Animation.BackEase in WpfGeneratedKnownTypes which means we should use the reflection to create the object.

![](https://user-images.githubusercontent.com/16054566/168504677-a5f87161-dac4-470e-a213-329b31f32db8.png)

**Comment by pchaurasia14** [comment]
While `System.Xaml.dll!System.Xaml.Schema.XamlObjectCreationFactory` can create object directly by `WpfGeneratedKnownTypes` help, it may limit the type of objects that can be created. With the current approach _(using reflection (albeit slow))_, `XamlObjectCreationFactory` can create user-defined objects as well. 

This may have been by design.

**Comment by lindexi** [comment]
@pchaurasia14 How about add the `System.Windows.Media.Animation.BackEase` to WpfGeneratedKnownTypes ?

**Comment by pchaurasia14** [comment]
@lindexi - Yes, we can and will take it up once the higher priority items are dealt with.

**Comment by Kuldeep-MS** [comment]
@lindexi - can you please add reproducer for the screenshot you attached in the issue? this will help me to better understand the context.

**Comment by lindexi** [comment]
@Kuldeep-MS  Thank you. This is not a bug. And I means that we should add the `System.Windows.Media.Animation.BackEase` to WpfGeneratedKnownTypes to improve performance. So I can not add the reproducer for the screenshot

**Comment by Kuldeep-MS** [comment]
Hi @lindexi,

would you like to contribute on this issue. For your reference similar issue has been fixed here https://github.com/dotnet/wpf/pull/7164

**Comment by lindexi** [comment]
@Kuldeep-MS  Yes, and should I wait the https://github.com/dotnet/wpf/pull/7164 merged?

**Comment by Kuldeep-MS** [comment]
@lindexi - Yes, please wait till https://github.com/dotnet/wpf/pull/7164 is merged. We will be taking https://github.com/dotnet/wpf/pull/7164 in upcoming CTP(s).

**Comment by lindexi** [comment]
@Kuldeep-MS Thank you. I will begin

**Comment by lindexi** [comment]
@Kuldeep-MS I create https://github.com/dotnet/wpf/pull/8039 and could you review it. I don't know if I wrote the right code
